src/db/fakeData.js

```diff
@@ -66,9 +66,27 @@ export const users = [
          week2: null,
          week3: null
        },
-         
+      attended: {
```

Comment:
The class attendance object should be like this:
,
        classAttendance: {
            'HTML/CSS': {
              week1: 'yes',
              week2: null,
              week3: null
            },
            'JS Quizz 1': {
              week1: null,
              week2: 'no',
              week3: null
            },
            'JavaScript 1': {
              week1: null,
              week2: null,
              week3: null
            },
            'JavaScript 2': {
             week1: null,
             week2: null,
             week3: null
           },
           'JavaScript 3': {
             week1: null,
             week2: null,
             week3: null
           }
         }

Is no need for that attended:{
0: yes
1: no
3: late
}

Reply:
Sadat has done these amendments in the next pull request.